typescript/mainLogic/services/service_context.ts

```diff
@@ -0,0 +1,157 @@
+/*package services*/ //TODO Get rid of this, not needed in typescript
+
+//import (
+	//"context" //TODO Don't need context for typescript, golang specific
+import * as core_api_bindings_js_grpc from '../../core_api_bindings/api_container_service_grpc_pb.js';
```

Comment:
I _think_ Typescript might use lowerCamelCase for package names (which we should conform to if so), though I'm not sure